### ISSUE TITLE
LayeredChart not working with mark_errorBar()

### ISSUE THREAD

**Issue by apahl** [altair-viz/altair#269]
Hi,
thanks a lot for this very promising plotting library.
I am interested in plotting scientific data and therefore usually need to use error bars.
I was very pleased to see that they are implemented, but I could not get them to work, even when using the workaround described in #180.
```Python
import pandas as pd
from altair import Chart

data = pd.DataFrame({'a': list('CCCDDDEEE'),
                     'b': [2, 7, 4, 1, 2, 6, 8, 4, 7]})

chart = Chart(data).mark_point().encode(
    x='a',
    y='mean(b)'
) + Chart(data).mark_errorBar().encode(
    x='a',
    y='min(b)', 
    y2='max(b)'
)
chart.data = data  # workaround described in issue #180
chart
```
Executing this cell in the Jupyter Notebook does not output anything.
Executing any of the charts individually does work.
Many thanks for your help.

Kind regards,
Axel

**Comment by jakevdp** [comment]
Hi,
I believe the current layering interface in Vega-Lite is still experimental, and these kinds of issues should be fixed in the 2.0 release. @kanitw – is that a fair statement?

**Comment by apahl** [comment]
Ok, thanks, then I will wait for that.
BTW, many thanks also for the very interesting [podcast](https://talkpython.fm/episodes/show/81/python-and-machine-learning-in-astronomy) interview.

**Comment by jakevdp** [comment]
Layering is fixed in 2.0, but errorBar will not be supported until a future release.

I'm going to close this, because the reported issue is no longer a problem.
